src/main/java/lotto/domain/ManualLottos.java

```diff
@@ -0,0 +1,22 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class ManualLottos {
+    private final List<Lotto> manualLottos;
```

Comment:
```suggestion
    private final List<String> manualLottos;
```
위와 같이 사용자가 입력한 값을 String으로 저장하는 역할만 담당해도 되지 않을까?
List<Lotto>로 구현하니 LottoTickets과의 다른 역할이 없을 수도 있겠다.